_snippets/supported-channels.mdx

```diff
@@ -0,0 +1,8 @@
+| Channel<br/><br/> | Content Style<br/><br/> | Custom Variables<br/>`{{handlebars}}` format |
+| --- | --- | :-: |
+| **Email** | HTML | ✅ |
+|  | Visual Editor | ✅ |
+| **SMS** | Text | ✅ |
+| **Chat** | Text | ✅ |
+| **In-App** | Text | ✅ |
+| **Push** | Text | ✅ |
```

Comment:
This has 20+ occurrences across the docs, and is likely to change as we support more channels, so I've created a snippet.